tests/integ/test_dataframe.py

```diff
@@ -2634,6 +2671,45 @@ def test_replace_with_coercion(session):
     with pytest.raises(ValueError) as ex_info:
         df.replace([1], [2, 3])
     assert "to_replace and value lists should be of the same length" in str(ex_info)
+    # Replace Decimal value with int
+    Utils.check_answer(
```

Comment:
it seems related to coercion in localtesting, you can temporarily skip the local testing by:

```python
def test_replace_with_coercion(session, local_testing_mode):
    # existing code
    # ...
    if local_testing_mode:
        # SNOW-xxxx local test gap
        return
    # your new test code
```